tests/IceRpc.Protobuf.Tests/ServiceTests.proto

```diff
@@ -3,9 +3,20 @@
 syntax = "proto3";
 option csharp_namespace = "IceRpc.Protobuf.Tests";
 
+import "google/protobuf/empty.proto";
+
 package icerpc.protobuf.tests;
 
 service MyWidget {}
 
 // Test with unusual name casing.
 service myOtherWidget {}
+
+
+service MyFirstWidget {
+	rpc OpFirst(google.protobuf.Empty) returns(google.protobuf.Empty);
```

Comment:
From https://protobuf.com/docs/language-spec#methods and https://protobuf.dev/programming-guides/proto2/#services, there is a space after `returns`.